src/main/org/apache/tools/ant/taskdefs/optional/junitlauncher/LegacyXmlResultFormatter.java

```diff
@@ -298,8 +298,7 @@ private void writeFailed(final XMLStreamWriter writer, final TestIdentifier test
                     writeAttribute(writer, ATTR_MESSAGE, message);
                 }
                 writeAttribute(writer, ATTR_TYPE, t.getClass().getName());
-                // write out the stacktrace
```

Comment:
Hello @tsmock, was it intentional to remove this comment? I think it should stay.

Reply:
No, it was not intentional. I'll add it back.